hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -359,10 +359,10 @@ estimateScriptsCost ::
   Either ErrCoverFee (Map RdmrPtr ExUnits)
 estimateScriptsCost pparams systemStart epochInfo utxo tx = do
   case result of
-    Left translationError ->
-      Left $ ErrTranslationError translationError
+    Left tranlationError ->
+      Left $ ErrTranslationError tranlationError
     Right units ->
-      Map.traverseWithKey (\ptr -> left $ ErrScriptExecutionFailed . (ptr,)) units
+      traverse (left ErrScriptExecutionFailed) units
```

Comment:
We might want to keep the `RdmrPtr` to have at least a chance to know which script failed.